javascript/core/objects/converting-to-and-from-json.md

```diff
@@ -0,0 +1,78 @@
+---
+author: EwaTrzemzalska
+
+type: normal
+
+category: fundamental
+
+tags:
+
+  - JSON
+  - introduction
+
+---
+# Converting to and from JSON
+
+---
+## Content
+
+`JSON.stringify()` method is used to convert a JavaScript object or value to a JSON string. 
+
+Example:
+
+```javascript
+const ourDog = {
+    name: 'Rocky',
+    age: 2,
+    'dog years': 14
+}
+
+console.log(JSON.stringify(ourDog));
+//prints {"name":"Rocky","age":2,"dog years":14}
```

Comment:
The `code-width` error is here because this line is longer than 44 chars (it's 48).

You can break it up into multiple lines or maybe just omit the text:

```
// {"name":"Rocky","age":2,"dog years":14}
```